src/test/java/uk/gov/hmcts/reform/unspec/callback/CallbackHandlerFactoryTest.java

```diff
@@ -86,4 +122,73 @@ void shouldDispatchCallback_whenValidCaseEvent() {
 
         assertEquals(RESPONSE, callbackResponse);
     }
+
+    @Test
+    void shouldBeTrue_whenEventIsAlreadyProcessed() {
```

Comment:
Should we put these tests in nested `IsEventAlreadyProcessed` class? Might be good to do the same for already existing `dispatch` method tests.

Reply:
method name is changed, so they fit as it is